operator/src/main/resources/scripts/stop-server.py

```diff
@@ -2,23 +2,197 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at
 # http://oss.oracle.com/licenses/upl.
 
+# This is called by the preStop hook script (stopServer.sh) that is specified
```

Comment:
Not a show-stopper, but a nice to have:

I just noticed that we're calling 'print' throughout this script instead of 'trace'.   'trace' decorates output with a timestamp that's in the same format as the Operator log's data format.    See the other bash scripts and the introspects .py script for an example of it's usage.   The idea is to import and use it this way:

```
# Include this script's current directory in the import path (so we can import traceUtils, etc.)
# sys.path.append('/weblogic-operator/scripts')

# Alternative way to dynamically get script's current directory
tmp_callerframerecord = inspect.stack()[0]    # 0 represents this line # 1 represents line at caller
tmp_info = inspect.getframeinfo(tmp_callerframerecord[0])
tmp_scriptdir=os.path.dirname(tmp_info[0])
sys.path.append(tmp_scriptdir)

from traceUtils import *

...

trace(string)

...

```
Caution:  I dont know if you can directly replace a 'print exc' with a 'trace(exc)' - doubt it.